frontend/src/store/auth-context.tsx

```diff
@@ -0,0 +1,18 @@
+"use client"
+
+import {createContext} from "react";
+
+export interface AuthContextProps {
+  userLogin: string | undefined;
+  setUserLogin: (login: string) => void;
+  logout: () => void;
+}
+
+const authContextDefaultValue: AuthContextProps = {
+  userLogin: undefined,
+  setUserLogin: () => {},
```

Comment:
Imo auth context should store `user` object, `login` and `logout` functions. Authentication part is subject to change, so we will see how things develop.